Plugins/SwiftBundlerCommandPlugin/SwiftBundlerCommandPlugin.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import PackagePlugin
+
+@main
+struct SwiftBundlerCommandPlugin: CommandPlugin {
+  /// This entry point is called when operating on a Swift package.
+  func performCommand(context: PluginContext, arguments: [String]) throws {
+    let bundler = try context.tool(named: "swift-bundler")
+
+    try run(command: bundler.path, with: arguments)
+  }
+}
+
+#if canImport(XcodeProjectPlugin)
+  import XcodeProjectPlugin
+
+  extension SwiftBundlerCommandPlugin: XcodeCommandPlugin {
```

Comment:
Is the conformance to XcodeCommandPlugin useful? Swift Bundler only ever works properly if there is a Package.swift file in the root of the project, and I think XcodeCommandPlugin is only required if Xcode is operating on an xcodeproj instead of a Swift package.

Reply:
Ah! I understand now, thanks for the insight. SwiftPM plugins are still brand new to me, when I had originally skimmed some information on it what I had thought I read is that when **Xcode** builds a **SwiftPM** package from within **Xcode** it uses that `XcodeCommandPlugin` entrypoint instead of the `CommandPlugin` entrypoint.

Reply:
I'd never seen XcodeCommandPlugin before 😅 took me a while to figure out what it did